system/datatypes/mailinglist.php

```diff
@@ -229,7 +230,7 @@ private function displayArchive() {
 				$date = '';
 			}
 			$row->addCell($date);
-			$buttons = $this->makeActionButton(__('read'), $mailing->getId());
+			$buttons = $this->makeActionButton(__('view'), $mailing->getId());
```

Comment:
This change does not seem to be mentioned in the commit message? I suspect it would be a separate commit even?